### DISCUSSION TITLE
UnitOfWorkManager内部异常导致应用终止

### DISCUSSION THREAD

**Discussion by lzy198845** [dotnetcore/FreeSql#813] [Q&A]
![image](https://user-images.githubusercontent.com/29939110/124083746-d1797880-da80-11eb-9ba7-8ee6c302f489.png)
 public class MyUnitOfWorkManager : UnitOfWorkManager
    {
        public MyUnitOfWorkManager() : base(GetFreesql())
        {
        }

        public static IFreeSql GetFreesql()
        {
            throw new Exception("内部异常");
        }
}

**Comment by lzy198845** [comment]
首先感谢叶师傅无私奉献，

**Reply by luoyunchong** [reply]
为什么异常会导致应用终止，你是什么程序？winform ,asp.net core。不至于吧。你写一个控制台，throw 一个异常。不try 住，不也一样终止。你应该try住他。
